qa/covid19_qa/qa.py

```diff
@@ -36,16 +37,16 @@ def answer_from_instances(instances: Iterator[Instance], qa_pipeline: Pipeline,
     else:
         return heapq.nlargest(top_k, answers, key=lambda a: a.sort_score)
 
+
+@lru_cache
```

Comment:
We cache up to 128 input parameter configurations (the default), which should be only diff questions. We can change this number later, or even make it an env var.

Reply:
You can test this by making the same request twice, and noting is way faster after the first time.